scripts/observe_manual_run_to_extract_checkpoints.py

```diff
@@ -1,10 +1,13 @@
+import os
 import sys
 from pathlib import Path
 
 import numpy as np
 from tminterface.client import Client, run_client
 from tminterface.interface import TMInterface
 
+if not os.path.exists("maps"):
```

Comment:
maybe use pathlib existing `mkdir` method since it is used instead of using `os`? https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir